ccp/modules/exporter-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: "3.7"
+
+services:
+  exporter:
+    image: docker.verbis.dkfz.de/ccp/dktk-exporter:latest
+    container_name: bridgehead-ccp-exporter
+    environment:
+      LOG_LEVEL: "INFO"
+      EXPORTER_API_KEY: "${EXPORTER_API_KEY}"
+      CROSS_ORIGINS: "https://${HOST}"
+      EXPORTER_DB_USER: "exporter"
+      EXPORTER_DB_PASSWORD: "${EXPORTER_DB_PASSWORD}" # Set in exporter-setup.sh
+      EXPORTER_DB_URL: "jdbc:postgresql://exporter-db:5432/exporter"
+      CLEAN_TEMP_FILES_CRON_EXPRESSION: "0 0 1 * * *"
+      TEMP_FILES_LIFETIME_IN_DAYS: "1"
+      CLEAN_WRITE_FILES_CRON_EXPRESSION: "0 0 2 * * *"
+      WRITE_FILES_LIFETIME_IN_DAYS: "30"
+      HTTP_RELATIVE_PATH: "/ccp-exporter"
+      HTTP_SERVLET_REQUEST_SCHEME: "https"
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.exporter_ccp.rule=PathPrefix(`/ccp-exporter`)"
+      - "traefik.http.services.exporter_ccp.loadbalancer.server.port=8092"
+      - "traefik.http.routers.exporter_ccp.tls=true"
+      # TODO: Add basic auth in Teiler UI to access exporter?
+      #- "traefik.http.routers.exporter_ccp.middlewares=auth"
+      - "traefik.http.middlewares.exporter_ccp_strip.stripprefix.prefixes=/ccp-exporter"
+      - "traefik.http.routers.exporter_ccp.middlewares=exporter_ccp_strip"
+    volumes:
+      - "bridgehead-exporter:/app/exporter-files"
+
+  exporter-db:
+    image: postgres:15.1-alpine
+    container_name: bridgehead-ccp-exporter-db
+    environment:
+      POSTGRES_USER: "exporter"
+      POSTGRES_PASSWORD: "${EXPORTER_DB_PASSWORD}" # Set in exporter-setup.sh
+      POSTGRES_DB: "exporter"
+    volumes:
+      - "bridgehead-exporter-db:/var/lib/postgresql/data"
+
+
+volumes:
+  bridgehead-exporter-db:
+    name: "bridgehead-exporter-db"
+  bridgehead-exporter:
+    name: "bridgehead-exporter"
```

Comment:
Please check if you can do without these volumes.

Reply:
If we remove these volumes, the exports cannot be downloaded after restarting the bridgehead. It must be taken into account, that the export files need to be at least temporally saved,  as the process works asynchronously: 1) Execution of a query is requested. 2) Export can be downloaded as soon as it is available. 

If an export file can be saved only until the bridgehead is restarted, I would suggest removing the volume "bridgehead-exporter".

Until we solve the problem of query persistence, we need a way of saving the queries. As soon as we have a final solution for that, I would suggest keeping the volume "bridgehead-exporter-db".